internal/pkg/githubactions/githubactions.go

```diff
@@ -122,14 +122,14 @@ func (ga *GithubActions) getFileSHA(filename string) (string, error) {
 		&github.RepositoryContentGetOptions{},
 	)
 
-	if resp.StatusCode == http.StatusNotFound {
-		return "", nil
-	}
-
 	if err != nil {
 		return "", err
 	}
 
+	if resp.StatusCode == http.StatusNotFound {
```

Comment:
should be in front of  line 125， or it will return error while file not found

Reply:
done